automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -9,19 +9,18 @@ import org.broadinstitute.dsde.rawls.model.WorkspaceName
 import org.broadinstitute.dsde.workbench.auth.AuthTokenScopes.billingScopes
 import org.broadinstitute.dsde.workbench.config.ServiceTestConfig
 import org.broadinstitute.dsde.workbench.leonardo.BillingProjectFixtureSpec._
-import org.broadinstitute.dsde.workbench.leonardo.TestUser.{Hermione, Ron}
+import org.broadinstitute.dsde.workbench.pipeline.TestUser.{Hermione, Ron}
```

Comment:
wait I think they might be failing because of this change, if the old tests are looking for the test user but the pipeline isn't run on the jenkins jobs they won't be able to get the authToken, right?

Reply:
looking at the stack trace closer, its failing on this line
`  implicit val ronTestersonAuthorization: IO[Authorization] = Ron.authorization()`
so I think that's the issue

Reply:
We have this situation where swat tests are running in Jenkins and Azure tests are shifting to GHA.
I decoupled `LeonardoAzureSuite` from `LeonardoSuite` to resolve this issue.
Once @jyang-broad has shifted Leo swat tests to GHA, we should start using the new pipeline for these tests as well.

Reply:
Let's discuss next week with @jyang-broad on timing of switching over from Jenkins to GHA.